scm-record/README.md

```diff
@@ -57,7 +57,6 @@ Here's some projects that don't use `scm-record`, but could benefit from integra
 
 Here are some features in the UI which are not yet implemented:
 
-- Select inner/outer element.
```

Comment:
Looks like the Keybindings section should be update with `left` and `right` now (and `j` and `l`)